tests/integration/testecbug.nim

```diff
@@ -0,0 +1,65 @@
+from pkg/libp2p import Cid, init
+import ../examples
+import ./marketplacesuite
+import ./nodeconfigs
+import ./hardhatconfig
+
+marketplacesuite "EC bug":
```

Comment:
Hmm... Can we rename this something more appropriate? What EC bug, what does it actually test, etc?